tests/unittests/graphTest.cpp

```diff
@@ -1155,4 +1155,33 @@ TEST(Graph, verifyConstantNoWriters) {
   EXPECT_DEATH(M.verify(), "");
 }
 
+/// Check that the verifier will complain if a constant and its
```

Comment:
Do we get a good signal-to-noise ratio from this unit test?

Will the testing benefits of this unit test outweigh the cost of having to maintain it?

Reply:
I have mixed feeling about this one.
Most of our verifier checks are untested but recently @jfix71 showed us a path how to check them and I liked it.
I don't expect this will need updating in the (near) future, but I don't have a strong opinion either way.
What do you think?

Reply:
I don't have a strong opinion about this specific PR. I just wanted to start this discussion that can guide us in future work.